test/unit_tests/braket/ocean_plugin/test_braket_sampler.py

```diff
@@ -62,6 +72,18 @@ def test_parameters(braket_sampler):
     assert isinstance(braket_sampler.parameters, FrozenDict)
 
 
+def test_advantage_parameters(advantage_braket_sampler):
+    expected_params = {
+        param: ["parameters"] for param in BraketSolverMetadata.DWAVE["parameters"].values()
+    }
+    del expected_params["beta"]
+    del expected_params["chains"]
+    del expected_params["postprocessingType"]
```

Comment:
Why do these need to be explicitly deleted? They shouldn't be present in the first place.

Reply:
They are deleted from `expected_params` which is constructed from the list of parameters in the `BraketSolverMetadata` class